book/chapters/gridmet.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
the ipython json is too long here in github to review correctly. I'll just type it out here instead.

<img width="638" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/cda3be8b-ac13-4d4b-a71d-35c45eb1641c">
I dont think we need map visualization logic here in gridmet section.
This needs to be at the last but 1 chapter before conclusion.

Reply:
done